lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -45,6 +45,10 @@ function dosomething_signup_form($form, &$form_state, $nid, $label = "Sign Up")
 
   // Check query string for signup source value.
   if ($source = dosomething_signup_get_query_source()) {
```

Comment:
@DFurnes I don't think the signups table has a `source_details` column but it does have a `source` column! Should I put it there instead?

Reply:
an example of something that might be in that column is this:
```
{"campaign_id":"6uFn4194vCOYmoGm88EEwK","legacy_campaign_id":"8064","legacy_campaign_run_id":"8065"}
```

Reply:
Oh, bummer. I wonder if we should add a `source_details` there for consistency?

Reply:
I think that makes sense! I can work on that now and then update this to populate `source_details` instead of `source` :)

Reply:
Thanks @chloealee!! 👍

Reply:
Of course - thank you for flagging this, @DFurnes ! Looking into a few other records that have the `details` field populated, I'm seeing some other trends: 

```
{"hostname":"dosomething.turbovote.org","referral-code":"***","partner-comms-opt-in":"","created-at":"2018-05-20T22:02:42Z","updated-at":"2018-05-20T22:02:57
```

```
affiliate-opt-out
```

I'm wondering if I should just change `details` to `source_details` instead of adding another column? This would probably affect some other apps like chompy though cc @sbsmith86

Reply:
I think we use `details` as more of a catch-all bucket, so it probably should stay as is.

Reply:
ah that makes sense! [Here's the PR](https://github.com/DoSomething/rogue/pull/744) to add the `source_details` column :)